mantidproject/mantid
# Make Vates algorithms accessible from python

Description of work.

The Vates algorithms `LoadVTK` and `SaveMDWorkspaceToVTK` were only available within MantidPlot. This moves them to a separate shared library that is accessible from vanilla python.

**To test:**



1. start mantidpython (NOT MantidPlot)
2. run `from mantid.simpleapi import *`
3. use autocomplete to check if the SaveMDWorkspaceToVTK function is available.

There is No Github issue associated with this pull request

**Release Notes** 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
